cmd/keycloak-operator/main.go

```diff
@@ -29,13 +30,14 @@ var (
 	resyncFlag *int = new(int)
 )
 
-func setFlags() {
-	flag.IntVar(resyncFlag, "resync", 7, "change the resync period")
+func init() {
+	flagset := flag.CommandLine
+	flagset.IntVar(resyncFlag, "resync", 7, "change the resync period")
+	flagset.Parse(os.Args[1:])
```

Comment:
The operator flags weren't being parsed properly before, so things like this weren't working:

```
operator-sdk up local --operator-flags="--resync=2" --namespace shared
```